### ISSUE TITLE
Small question

### ISSUE THREAD

**Issue by martinnnnnn** [nlohmann/json#1094]
Hi,
Thanks for the library, it's really easy to setup and use :) I only have one small question :
Is it possible to set pretty print when printing a json object without setting it for arrays in the object ? I end up with 
`"obj1": {`
`        "key1": "value1",`
`        "key2": [`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0,`
   `         0.0`
   `     ]`
   ` },`
(there should be tabs but I couldn't get the code insertion to do what I want ^^')
which doesn't look very good.. Any way I can set the array to print in 1 line ? I could probably iterate thought each objects and check when there is an array and print it differenlty but that seems like a lot of work !
Thanks :)

**Comment by nlohmann** [comment]
The pretty-printer is currently not so flexible. The set indentation is used for both objects and arrays. You can, however, set the indentation character to a tabs, see https://nlohmann.github.io/json/classnlohmann_1_1basic__json_a5adea76fedba9898d404fef8598aa663.html#a5adea76fedba9898d404fef8598aa663

**Comment by martinnnnnn** [comment]
Thanks for your quick response ! 
If I were to look into it and add the possibility to set different identations for objects and arrays, would you like me to create a pull request or is it something a feature you don't want added to the library ?
Have a good day, thanks again for the library it's awesome :)

**Comment by nlohmann** [comment]
I would like to discuss the interface first. Right now, the dump function already has 3 parameters, and I would like to avoid a complete bloat of parameters for use cases that are very rare.

**Comment by martinnnnnn** [comment]
Yeah, I managed to modify it to get the result I wanted but the interface
got pretty ugly. I ended up simply removing the pretty print for arrays
because I like it better this way but this is just my personal taste :) I
agree with you, it's a rare use case and the modification required is
really small so it might not be such a good idea to change anything...
